Nikita Korzhavin/flex/cubes.css

```diff
@@ -0,0 +1,479 @@
+  box-sizing: border-box;
+}
+
+.card__text {
+  color: #fff;
+  font-family: 'AvenirNext-Bold';
+}
+
+.card__text_size_m {
+  font-size: 1rem;
+}
+
+.card__img {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: end;
+  -ms-flex-pack: end;
+  justify-content: flex-end;
+}
+
+.card_bg-color_06b3db {
+  background-color: #06b3db;
+}
+
+.card_bg-color_4ed486 {
+  background: #4ed486;
+}
+
+.card_bg-color_6139f6 {
+  background: #6139f6;
+}
+
+.card_bg-color_e3b63d {
+  background: #e3b63d;
+}
+
+.card_bg-color_bd54cd {
+  background: #bd54cd;
+}
+
+.card_bg-color_bd3559 {
+  background: #bd3559;
+}
+
+.card__top {
+  display: none;
+}
+
+.card__bottom {
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
+  flex: 1;
+}
+
+.card__info-wrapper {
+  width: 100%;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: space-between;
+}
+
+.card__info-day {
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
+  flex: 1;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.card__text_size_s {
+  font-size: 0.814rem;
+}
+
+.card__img_size_s {
+  height: 30px;
+  min-width: 25px;
+}
+
+.card__img_opacity {
+  opacity: 0.45;
+}
+
+.card__text_flex {
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
+  flex: 1;
+  -webkit-box-ordinal-group: 2;
+  -ms-flex-order: 1;
+  order: 1;
+  margin-left: -15px;
+}
+
+.card__text_margin_l {
+  margin-left: 0px;
+  font-size: 0.614rem;
+}
+
+@media (min-width: 576px) {
+  .bg1 {
+    background: url('./images/bg_1.png') no-repeat;
+    height: 500px;
+    width: 600px;
+  }
+
+  .bg2 {
+    background: url('./images/bg_2.png') no-repeat;
+    height: 500px;
+    width: 600px;
+
+    display: -webkit-box;
+
+    display: -ms-flexbox;
+
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+  }
+
+  .widget {
+    max-width: 400px;
+  }
+}
+
+@media (min-width: 768px) {
+  .widget {
+    max-width: 500px;
+  }
+}
+
+@media (min-width: 992px) {
+  .bg1 {
+    background: url('./images/bg_1.png') no-repeat;
+    height: 600px;
```

Comment:
Why do you use fixed height and width for instead of stretching everyting to fullscreen? It is crucial in terms of responsiveness.

Reply:
To say the truth I thought that that background isn't necessary part of the widget and so i didnt't pay much attention to its look 
so i' ll fix that

Reply:
fixed